tmt/convert.py

```diff
@@ -170,6 +173,19 @@ def write_markdown(path, content):
         raise ConvertError(f"Unable to write '{path}'.")
 
 
+def add_bug(bug, data):
+    """ Add relevant bug into data under the 'link' key """
+    new_link = dict(verifies=f"{BUGZILLA_URL}{bug}")
```

Comment:
Are you sure `verifies` relation is always applicable for found bugs? I can imagine that in Nitrate there might be linked bugs which are somehow 'related' to the test case but aren't verified by it.
But that might just be couple of cases and not worth considering.

Reply:
Well, yeah, your're rigth we cannot be always sure about the proper relation. I would suggest to default to the most frequent one and leave the rest upon users. Or should we just default to generic `relates` instead? @Athwale, what do you think?

Reply:
I can not say how many cases have more bugs attached to them which are not verified by the case (I know about a few, for example selinux bugs are sometimes added to tests that discovered them but a separate selinux test is written for them later), "relates" seems more reasonable to me. 
At least in my case this information is mainly used to automatically set appropriate flags in a bug when uploading a new test as it is now with Makefiles and to see what are the original reproducers when fixing a test. If this keyword is not planned to be used for anything bigger I think it does not really matter what is is called.

Reply:
My PoV: Import is a helper tool, human should review it. With that said, tmt has no information about the relation between bug and test, nor Makefile does. Traditionally it is 'verifies' for Makefile bugs but probably relates for Nitrate (goal was just to mark connection, not necessary verifies. E.g. no results available because of another bug - you still connect bug to case so in the future you have easier review.).

So 'relates' seems safer. User should review meta data anyway. And if they doesn't it is better to report relation as 'relates' than claim it really 'verifies'.

Reply:
Agreed, `relates` seems to be a better default. Fixed in 52290e9. Thanks for the feedback.